h/subscribers.py

```diff
@@ -31,22 +31,20 @@ def add_renderer_globals(event):
 
 
 @subscriber(AnnotationEvent)
-def send_reply_notifications(
-    event,
-    get_notification=reply.get_notification,
-    generate_mail=emails.reply_notification.generate,
-    send=mailer.send.delay,
```

Comment:
This must be pretty old code because we haven't used this style of injecting dependencies in a long time. Since you can just patch imports in Python, it's not necessary to do this